guides/browser-support.md

```diff
@@ -114,6 +114,33 @@ browsers:
 - iOS Safari
 - Chrome for Android
 
+#### Manual testing
+
+We use a mix of Saucelabs and [localtunnel](https://localtunnel.github.io/www/)
+to manually test our site. Be sure to review both the testing and accessibility
+guides when reviewing sites locally.
```

Comment:
Should there be links to the guides here or a blurb on where to find them? I feel like someone joining the project could read this and be like "where do I find those?"